packages/node-sdk/paima-db/docker-pgtyped.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -eu
```

Comment:
If you think running this docker image is the easiest way to update the DB types, probably it's best to update the `paima-db` README